test/prob/test_fixture_distr.hpp

```diff
@@ -251,8 +252,8 @@ class AgradDistributionTestFixture : public ::testing::Test {
                                         Scalar3, Scalar4, Scalar5>(p0, p1, p2,
                                                                    p3, p4, p5);
 
-      EXPECT_TRUE(reference_logprob_false - logprob_false
-                  == reference_logprob_true - logprob_true)
+      EXPECT_NEAR(value_of_rec(reference_logprob_false - logprob_false),
+                  value_of_rec(reference_logprob_true - logprob_true), 1e-12)
```

Comment:
I might have missed this in the other issues, but why would we no longer expect the log-probabilities to be identical between the ```propto=true``` and ```propto=false``` conditions?

Reply:
It's a floating point comparison thing. Different code runs with `propto=true`/`propto=false`, so things can round differently.

Here was the error from Jenkins:
```
./test/prob/test_fixture_distr.hpp:256: Failure
Value of: value_of_rec(reference_logprob_false - logprob_false) == value_of_rec(reference_logprob_true - logprob_true)
  Actual: false
Expected: true
Proportional test failed at index: 1
  reference params: (1.2, 0.29999999999999999, 2)
  current params:   (-1, 0.20000000000000001, 0.25)
  ref<true> = -1.4364978919949096
  cur<true> = -1.4901106714054309
  ref<false> = -2.129645072554855
  cur<false> = -2.1832578519653758
```

And then if you compute the difference in R:
```
> -1.4364978919949096 + 1.4901106714054309 -
  (-2.129645072554855 + 2.1832578519653758)
[1] 4.440892e-16
```

I just picked 1e-12 cause I think it'll catch real errors and avoid anyone worrying about finickiness like this.